crates/runtime/src/component/dataset/mod.rs

```diff
@@ -215,6 +215,34 @@ impl Display for ReadyState {
     }
 }
 
+/// Controls whether the federated table periodically has its availability checked.
+#[derive(Debug, Clone, Copy, PartialEq, Default)]
+pub enum AvailabilityMonitor {
+    /// The dataset is checked for availability if it isn't accelerated.
+    #[default]
+    Default,
```

Comment:
I don't entirely understand what `AvailabilityMonitor::Default` means. I assume its just `AvailabilityMonitor::Enabled` or similar?

Reply:
Its not exactly `Enabled` because the current behavior only enables it for non-accelerated datasets, so this is saying we're choosing the "Default" behavior that happens today.